open_seq2seq/parts/convs2s/utils.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
```

Comment:
Why is this file necessary? We have all these functions under parts/transformer/utils.py 
You should just re-use "get_padding" and "get_padding_bias" from there.